src/main/java/com/ibm/cloud/sdk/core/security/AbstractToken.java

```diff
@@ -23,7 +23,7 @@
 
   // This field will be used to indicate that the most recent interaction with the token server
   // resulted in an error.
-  private Throwable exception;
+  private transient Throwable exception;
```

Comment:
This modifier will cause GSON to ignore this field for marshaling/unmarshaling purposes, which means that GSON will no longer try to call "setAccessible()" for the Throwable.detailedMessage field.